mysite/templates/ct/partial_actions.js.html

```diff
@@ -1,182 +1,193 @@
 <script type="text/javascript">
-    (function () {
```

Comment:
Almost all ``partial_actions.js.html`` was changed. It is very hard to understand where is your new code here.
Please make a rebase on latest base brach (fix_issue#93) to fix this.